accounts/tests.py

```diff
@@ -638,10 +639,7 @@ def test_select_uploaded_files_to_describe(self):
         self.client.login(username='testuser', password='testpass')
         user_upload_path = settings.UPLOADS_PATH + '/%i/' % user.id
```

Comment:
I note that this will create a temp folder and not clean up after the test. Is this something that we could clean up in a teardown method, or at the end of the test?

Reply:
Addressed in 1a0e0b5642771df0cc1c05aef62c300d1b15d068